bin/resty

```diff
@@ -475,6 +482,8 @@ Options:
     --main-include path Include the specified file in the nginx main configuration block
                         (multiple instances are supported).
 
+    --ns IP             Specify the nameserver(s) (multiple instances are supported)
```

Comment:
Shouldn't this be:
```
Specify a custom name server (multiple instances are supported)
```

- added "custom" to imply that by setting this field, the ones from `resolv.conf` won't be considered
- switched to singular, because this option only takes 1 ns, but can indeed be repeated
- "name server" instead of "nameserver" because it's the commonly used wording (including in the Nginx documentation of the `resolver` directive.

Just a thought, I don't see this as a blocker, just maybe an improvement :)